oak-store-spi/src/main/java/org/apache/jackrabbit/oak/spi/state/AbstractNodeState.java

```diff
@@ -50,6 +52,7 @@
  * alternatives.
  */
 public abstract class AbstractNodeState implements NodeState {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractNodeState.class.getName());
```

Comment:
Even though the logging is protected with an `isDebugEnabled()` - as this is quite a core class I'd like to better understand - is this really needed, and what for? could we do without this?

Reply:
This logging will help us in case we are failing in verification during DGC.
Since these logs would only execute in case nodes are not equal and debugging is enabled.